src/app/profile/spaces/spaces.component.ts

```diff
@@ -26,7 +26,7 @@ export class SpacesComponent implements OnInit {
     private spaceService: SpaceService,
     private logger: Logger,
     private contexts: Contexts) {
-      this.contexts.current.subscribe(val => this.context = val);
+    this.contexts.current.subscribe(val => this.context = val);
```

Comment:
I'm not sure why your IDE changed this, but it should be indented. Not a priority, just noting it. 
It looks like the line below should have been indented instead.